about.html

```diff
@@ -7,59 +7,118 @@
+     <a class="Products"href="#">Products</a>
+     <a class="Blog"href="#">Blog</a>
+     <a class="Contact"href="#">Contact</a>
+    </div>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    </nav>
 
-    Learn More
+    <section class="jumbo-about">
+        <img src="img/jumbo-about.png" alt="jumbo-about">
+        </section>
 
 
 
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+        
+</header>
+   
+<div class="sometexttop">
+
+<h2>About</h2>
+
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    </p>
+
+    </div>
+
+   
+<section class="middle-content">
+  <div class="boxes">
+    <div class="box1 test">
+      <img src="img/about-plan.png" alt="strategy">
+      
+      <h2>Strategy</h2>
+      
+      <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+      <a href="#">Learn More</a>
+    </div>
+   <div class="box2 test">
+          <img src="img/about-working.png" alt="strategy">
+        
+        <h2>How We Work</h2>
+        
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        <a href="#">Learn More</a>
 
-    Learn More
+    </div>
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <div class="box3 test">
 
-    Learn More
+        <img src="img/about-office.png" alt="strategy">
+        
+        <h2>Places We Work</h2>
 
-    Let's Work Together
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        <a href="#">Learn More</a>
+
+    </div>
+
+    <div class="box4 test">
+      <img src="img/about-meeting.png" alt="strategy">
+        
+        <h2>Collaboration</h2>
         
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+        <a href="#">Learn More</a>
+    </div>
+
+  </div>
+
+</section>
+
+<div class="sometextbottom">
+
```

Comment:
For this div could it have been changed to a more semantic portion of code like a section or an article?

Reply:
yes, it can be replaced by  a section element